src/components/Form.js

```diff
@@ -172,8 +173,20 @@ class Form extends React.Component {
     validate(values) {
         FormActions.setErrors(this.props.formID, null);
         FormActions.setErrorFields(this.props.formID, null);
+
+        // Run any validations passed as a prop
         const validationErrors = this.props.validate(values);
 
+        // Validate the input for html tags. It should supercede any other error
+        _.each(values, (inputValue, inputID) => {
+            // Check for HTML tags in string values
+            if (!inputValue || (_.isString(inputValue) && inputValue.search(CONST.VALIDATE_FOR_HTML_TAG_REGEX) === -1)) {
```

Comment:
Sorry, I think the logic I gave is wrong actually :ohnothing: we want to return early and avoid reporting the error when:

- there's no input value at all
- it's not a string
- it is a string that has no html

so 
```js
if (!inputValue || !_.isString(inputValue) || inputValue.search(CONST.VALIDATE_FOR_HTML_TAG_REGEX) === -1)
```
if we can pass non-strings with the current example they would fail to return early because

- it has a value
- it's not a string (`_.isString(inputValue) && inputValue.search(CONST.VALIDATE_FOR_HTML_TAG_REGEX) === -1)` short circuits to false)
- html detected falsely

Reply:
Ah, makes sense. Good catch.